packages/react/collapsible/src/Collapsible.tsx

```diff
@@ -201,6 +208,7 @@ const CollapsibleContentImpl = React.forwardRef<
       style={{
         [`--radix-collapsible-content-height` as any]: height ? `${height}px` : undefined,
         [`--radix-collapsible-content-width` as any]: width ? `${width}px` : undefined,
+        animation: context.isAnimationDisabled ? 'none' : undefined,
```

Comment:
Bit of an edge but should this be placed after the spread to account for mount animations supplied via `style` prop?

Also wondering if it's worth popping a quick note to explain why we're overriding this.

Reply:
We always allow consumer props to override our stuff so if they're supplying styles the assumption is that they are purposefully trying to override it I believe.

> Also wondering if it's worth popping a quick note to explain why we're overriding this.

Good shout

Reply:
> the assumption is that they are purposefully trying to override it I believe?

Yeah that's probably more likely tbh